__tests__/endpoints.test.js

```diff
@@ -68,15 +71,163 @@ describe("app", () => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              votes: 4,
+            })
+          );
+        });
+    });
+    describe("ERRORS", () => {
+      test("status 400, responds with error message if req body is an empty object", () => {
+        const reviewUpdate = {};
+        return request(app)
+          .patch("/api/reviews/2")
+          .expect(400)
+          .send(reviewUpdate)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("Bad Request");
+          });
+      });
+      test("status 400, responds with error message if update is of incorrect type", () => {
+        const reviewUpdate = { inc_votes: "i am a string, not a number!" };
+        return request(app)
+          .patch("/api/reviews/2")
+          .expect(400)
+          .send(reviewUpdate)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("Bad Request");
+          });
+      });
+    });
+  });
+  describe("GET /api/users", () => {
+    test("status 200, responds with an array of user objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body: { users } }) => {
+          expect(users).toBeInstanceOf(Array);
+          users.forEach((user) => {
+            expect(user).toEqual(
+              expect.objectContaining({
+                username: expect.any(String),
+                name: expect.any(String),
+                avatar_url: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+    describe("ERRORS", () => {
+      test("status 404, responds with error message when passed route that does not exist", () => {
+        return request(app)
+          .get("/api/uzerz")
+          .expect(404)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("Not Found");
+          });
+      });
+    });
+  });
+  describe("GET /api/reviews", () => {
+    test("status 200, responds with an array of review objects with the correct properties", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews).toBeInstanceOf(Array);
+          reviews.forEach((review) => {
+            expect(review).toEqual(
+              expect.objectContaining({
+                owner: expect.any(String),
+                title: expect.any(String),
+                review_id: expect.any(Number),
+                category: expect.any(String),
+                review_img_url: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                review_body: expect.any(String),
+                designer: expect.any(String),
+                comment_count: expect.any(String),
+              })
+            );
+          });
+          expect(reviews).toBeSortedBy("created_at", {
+            coerce: true,
+            descending: true,
+          });
```

Comment:
This order check isn't part of your test description. Consider separating this into it's own test